common/lib/xmodule/xmodule/js/src/video/09_events_plugin.js

```diff
@@ -142,7 +142,8 @@
             log: function(eventName, data) {
                 var logInfo = _.extend({
                     id: this.state.id,
-                    code: this.state.isYoutubeType() ? this.state.youtubeId() : 'html5'
+                    // eslint-disable-next-line no-nested-ternary
+                    code: this.state.isYoutubeType() ? this.state.youtubeId() : this.state.canPlayHLS ? 'hls' : 'html5'
```

Comment:
Created ticket for edx-analytics-pipeline https://openedx.atlassian.net/browse/TNL-6809 and  @sstack22 needs to confirm who'll pick this up along with warning the research community.